### DISCUSSION TITLE
Neo NAS-AB02B2 No converter available....

### DISCUSSION THREAD

**Discussion by skwat75** [Koenkk/zigbee2mqtt#20278] [Devices]
Hi,

I use a NAS-AB02B2 with zigbee2mqtt version 1.34.0 (herdsman 15.130.1) key sonoff P (20230507) 
I can drive the device but i 've got a lot of error like below in the log

I don't' understand, Neo NAS-AB02B2 is supposed to be supported.

Regards

Error 2023-12-18 19:41:02No converter available for 'battery_low' ("")
Error 2023-12-18 19:41:02No converter available for 'get' 'alarm' ()
Error 2023-12-18 19:41:02No converter available for 'get' 'melody' ()
Error 2023-12-18 19:41:02No converter available for 'get' 'duration' ()
Error 2023-12-18 19:41:02No converter available for 'get' 'volume' ()
Error 2023-12-18 19:41:02No converter available for 'battpercentage' ("")

**Comment by Psytoshgen** [comment]
same

**Reply by skwat75** [reply]
Hi,

Are you using zigbeelinker with jeedom ?
